src/Eagle.hpp

```diff
@@ -16,7 +16,7 @@ typedef struct
 
 
 //Variable
-typedef struct
+typedef struct EagleVariable
```

Comment:
I had to introduce a type, anonymous structs can't be forward declared in other headers.